distr_tracing/distr_tracing_otel/distr-tracing-otel-forwarding.adoc

```diff
@@ -0,0 +1,136 @@
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-forwarding"]
+= Forwarding traces to TempoStack by using OpenTelemetry Collector
+include::_attributes/common-attributes.adoc[]
+:context: distr-tracing-otel-forwarding
+
+To configure log forwarding to the TempoStack, you deploy and configure an OpenTelemetry collector.
```

Comment:
This should be either.

To configure forwarding Telemetry data..
or
To configure forwarding traces...